### ISSUE TITLE
PHP 8 support, please?

### ISSUE THREAD

**Issue by burzum** [prooph/event-store-http-client#56]
> prooph/event-store-http-client dev-master requires php ^7.4 -> your php version (8.0.6) does not satisfy that requirement.

The same problem is also present in the regular (non HTTP) client:

> prooph/event-store-client

**Comment by designermonkey** [comment]
I think it's fair to ask whether this package is supported now and will be recieving updates like this.

**Comment by prolic** [comment]
@designermonkey yes, PHP 8 will be supported. Feel free to submit a pull request, my time for open source development is limited.

**Comment by designermonkey** [comment]
I'll do my best to see if I can help :)

Slight tangent question, is the EventStore client supported still also?

**Comment by prolic** [comment]
even more so
